src/scripts/main.js

```diff
@@ -1,11 +1,24 @@
 'use strict';
 
 function waitFor(element, eventName) {
-// write your code here
+  const promise1 = new Promise((resolve) => {
+    element.addEventListener(eventName, () => {
+      resolve(
+        `It was ${eventName}
+       on the element: ${element.nodeName}, id: ${element.id}.`
```

Comment:
```suggestion
        `It was ${eventName}`
          + `on the element: ${element.nodeName}, id: ${element.id}.`
```

Reply:
Oleksandr, hi! Thank you for review and advice!